Docs/CommonErrors.md

```diff
@@ -0,0 +1,70 @@
+# Common Wallet Library Errors
+
+We return a Result object for every external interface that could have the potential to throw. Each public interface and what error could occur is outlined here.
+
+## Outer Errors
+### `VerifiedIdClient`
+
+`createVerifiedIdRequest`
+- `HttpError`:
+    - Code
+    - Retryable
+- `UnsupportedProtocolError`
+   - inner error with exact error
+   - Retryable: false
```

Comment:
Retryable: false on a lot of non network calls seems a bit excessive unless every error we return has a shared supertype with this property, which in itself seems like encouraging a pattern of "just check retryable and if its false, present an 'uh-oh' message" which is not the experience we want end users to have.